packages/terra-avatar/src/common/Avatar.module.scss

```diff
@@ -0,0 +1,178 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-clip: padding-box; // Stops background color from bleeding through border on IE.
+    border: var(--terra-avatar-border, solid 1px #fff);
+    border-radius: 50%;
+    color: var(--terra-avatar-color, #fff);
+    display: inline-flex;
+    height: var(--terra-avatar-initial-height, 3.286em);
+    justify-content: center;
+    line-height: 0;
+    min-height: 1.143em;
+    min-width: 1.143em;
+    overflow: hidden;
+    position: relative;
+    vertical-align: middle; // Fixes gap below image variant
+    width: var(--terra-avatar-initial-width, 3.286em);
```

Comment:
nit: I'd drop `initial` from this variable name.

Reply:
updated https://github.com/cerner/terra-core/pull/2061/commits/d9eca3475f6642229e82b08f42a7627a1548c481